src/App.js

```diff
@@ -1,24 +1,92 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import Contacts from './contacts';
+
+function ShowContacts() {
+  const [contacts, setContacts] = useState(Contacts.splice(0, 5));
+
+  const randomContactHandler = () => {
+    const contactCopy = [...contacts];
+    const getRandomContact = () =>
+      Contacts[
+        Math.floor(
+          Math.random() * (Contacts.length - (contactCopy.length - 1)) +
```

Comment:
Something is off in this logic.
Imagine: there are 10 Contacts, and 15 contacts in contactCopy. A Math.random would be multiplied with a negative number. Right?

Reply:
Please check the next comment. Taking that comment into account, you can retrieve the actor randomly quite a bit easier.